### DISCUSSION TITLE
Multi Drive Support

### DISCUSSION THREAD

**Discussion by WolfoTheWolf** [pelican-dev/panel#40] [Ideas]
- This would allow using multiple drives. Instead of needing to run RAID or combined drives via software it would be really nice if we could specify specific drives for servers, example, NODE 1, can have access to drive 1 - 4 on that available system if configured, thus it could auto rollover to the other drives IF they are full, or we can manually set an active drive to be used so we don't have to have game server files spread across two drives. Would like this to be a front end configurable thing as well.

**Comment by lancepioch** [comment]
Thanks, I believe mounts can already accomplish this!
